primitives/api/src/lib.rs

```diff
@@ -354,30 +358,33 @@ pub enum InitializeBlock<'a, Block: BlockT> {
 /// Parameters for [`CallApiAt::call_api_at`].
 #[cfg(feature = "std")]
 pub struct CallApiAtParams<'a, Block: BlockT, C, NC, Backend: StateBackend<HashFor<Block>>> {
-	/// A reference to something that implements the [`Core`] api.
-	pub core_api: &'a C,
-	/// The block id that determines the state that should be setup when calling the function.
-	pub at: &'a BlockId<Block>,
-	/// The name of the function that should be called.
-	pub function: &'static str,
-	/// An optional native call that calls the `function`. This is an optimization to call into a
-	/// native runtime without requiring to encode/decode the parameters. The native runtime can
-	/// still be called when this value is `None`, we then just fallback to encoding/decoding the
-	/// parameters.
-	pub native_call: Option<NC>,
-	/// The encoded arguments of the function.
-	pub arguments: Vec<u8>,
-	/// The overlayed changes that are on top of the state.
-	pub overlayed_changes: &'a RefCell<OverlayedChanges>,
-	/// The cache for storage transactions.
-	pub storage_transaction_cache: &'a RefCell<StorageTransactionCache<Block, Backend>>,
-	/// Determines if the function requires that `initialize_block` should be called before calling
-	/// the actual function.
-	pub initialize_block: InitializeBlock<'a, Block>,
-	/// The context this function is executed in.
-	pub context: ExecutionContext,
-	/// The optional proof recorder for recording storage accesses.
-	pub recorder: &'a Option<ProofRecorder<Block>>,
+    /// A reference to something that implements the [`Core`] api.
```

Comment:
Spaces!

Reply:
Leftover from the rescue after running `cargo fmt`